bottlecap

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+
+# bottlecap, 'cause it's kinda like cork... get it?
+
+usage() {
+	echo "usage:"
+	echo "bottlecap [--dev] [--dry-run] [--runtime $runtime] [--build-dir $build_dir]  [--container $container] assemblerargs..."
+	echo "  --dev: rebuild and reinstall mantle, the build scripts, and the ostree-releng-scipts before running coreos-assembler"
+	echo "  --dry-run: do not run CoreOS Assembler; print what commands would be executed instead"
+	echo "  --runtime: (auto) specify whether to use podman or docker. Defaults to podman if it is in \$PATH"
+	echo "  --build-dir: Where to put build artifacts, defaults to the current directory"
+	echo "  --container: which coreos-assembler container to use"
+	echo "  --help: print this help message"
+}
+
+dev=0
+dryrun=0
+help=0
+runtime="podman"
+command -v podman > /dev/null || runtime="docker"
+build_dir="$(pwd)"
+container="quay.io/cgwalters/coreos-assembler"
+
+rc=0
+TEMP=$(getopt -o 'dr:b:c:h' --long 'dev,dry-run,runtime:,build-dir:,container:,help' -- "$@") || rc=$?
+if [ "$rc" -ne 0 ]; then
+	echo "Bad arguments. getopt failed."
+	usage
+	exit 1
+fi
+
+eval set -- "$TEMP"
+
+while :
+do
+	case "$1" in
+		"-d"|"--dev")
+	  		dev=1
+			shift
+			;;
+		"--dry-run")
+			dryrun=1
+			shift
+			;;
+		"-r"|"--runtime")
+			shift
+			runtime="$1"
+			shift
+			;;
+		"-b"|"--build-dir")
+			shift
+			build_dir=$(readlink -f "$1")
+			shift
+			;;
+		"-c"|"--container")
+			shift
+			container="$1"
+			shift
+			;;
+		"-h"|"--help")
+			help=1
+			shift
+			;;
+		--)
+			shift
+			break
+			;;
+		*)
+			echo "Error parsing args"
```

Comment:
not sure if this section is needed any longer since we are using `getopt`, but if we keep this let's spit that message to stderr